spec/services/ingest_iati_activities_spec.rb

```diff
@@ -6,366 +6,387 @@
   let!(:existing_programme) { create(:programme_activity, identifier: "GCRF-INTPART", organisation: beis) }
 
   describe "#call" do
-    it "creates 36 new projects" do
-      uksa = create(:organisation, name: "UKSA", iati_reference: "GB-GOV-EA31")
-      legacy_activities = File.read("#{Rails.root}/spec/fixtures/activities/uksa/real_and_complete_legacy_file.xml")
-
-      service_object = described_class.new(delivery_partner: uksa, file_io: legacy_activities)
+    context "UKSA activities" do
```

Comment:
Tricky to follow the diff in this file to view the move versus the new test. I presume it is only the last test that was added and everything else is an indent, despite GitHub guessing that lots of lines were added and removed?

Reply:
Yeah I think adding this new context has made the diff a nightmare to read, I will rejig it.